### DISCUSSION TITLE
v5.64.0

### DISCUSSION THREAD

**Discussion by sokra** [webpack/webpack#14708] [General]
# Features

* add `asyncChunks: boolean` option to disable creation of async chunks

# Bugfixes

* fix ProfilingPlugin for `experiments.backCompat: false`

# Performance

* avoid running regexp twice over the file list

<hr /><em>This discussion was created from the release <a href='https://github.com/webpack/webpack/releases/tag/v5.64.0'>v5.64.0</a>.</em>

**Comment by iamgreenintro** [comment]
@sokra I just stumbled upon your feature request and notice it got implemented. Congratulations!

Now here's my question: https://github.com/webpack/webpack/discussions/17935

What does your feature request actually propose? What are the asyncChunks refering to?
Are they refering to the entry points in the `entry` object setting? And what if there's only one entry object?
